### PR TITLE
Add gradio-client

### PR DESCRIPTION
**Pull request by Sclare87**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by Sclare87** [comment]
@conda-forge/help-python this is ready to review thanks

**Comment by Sclare87** [comment]
Some months already passed.
Please check this for review.

File: recipes/gradio-client/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+test:
+  imports:
+    - gradio_client
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: https://pypi.org/project/gradio/
```
Comment (synapticarbors) [inline recipes/gradio-client/meta.yaml:41]: This should probably be: https://pypi.org/project/gradio-client

Also consider adding a `dev_url: https://github.com/gradio-app/gradio`

**Review by synapticarbors — COMMENTED** [review]
Sorry that this PR was overlooked. I just had a small comment then I'd be happy to merge.

File: recipes/gradio-client/meta.yaml (hunk 1 above)
Comment (Sclare87) [reply recipes/gradio-client/meta.yaml:41]: Thank you. I'll make the changes.

**Pull request merged**